### ISSUE TITLE
Can't Save Global Set with Address Field

### ISSUE THREAD

**Issue by richhayler** [barrelstrength/craft-sprout-fields#95]
### Description
CP Error when adding an address field to a globalset and attempting to save. 

### Steps to reproduce
1. Create address field - setting the language and country to "Australia" and toggling to a single country
2. Add to a globalset (on a multi-site setup) and attempt to save the layout in the field layout designer

### Additional info
- Craft version: 3.3.15
- PHP version: 7.2.24
- Database driver & version:  MySQL 5.7.27

**Comment by BenParizek** [comment]
Thanks @richhayler.

We'll let you know when we have a fix.

**Comment by andrelopez** [comment]
Hi @richhayler 
We're unable to recreate this error. Could you please recreate the error and send back to us a zip file of the following folder?
`storage/logs`
Please use support@barrelstrengthdesign.com

**Comment by BenParizek** [comment]
I've tested this again and been able to recreate it. Working on a fix.

**Comment by BenParizek** [comment]
Just a note until I get back to this. This bug has to do with running validation in the AddressFieldTraig::normalizeValue method:

``` php
if (!$addressModel->validate(null, false)) {
        return $value;
}
```

May need to revisit the logic of why validation is taking place in that method.

**Comment by BenParizek** [comment]
@richhayler This was a deeper issue than it appeared. It should be resolved in the latest release v3.5.1 which also includes several other updates that will help the address field play nicer with Drafts and Revisions.
